panning-view/src/main/java/com/maelstrom/panning/PanningImageView.java

```diff
@@ -26,9 +26,9 @@ public PanningImageView (Context context, AttributeSet attr, int defStyle) {
 		super (context, attr, defStyle);
 		if (!isInEditMode()) {
 			readStyleParameters(context, attr);
+			mAttacher = new PanningImageViewAttacher (this, mPanningDurationInMs);
```

Comment:
sorry, i didn't realize this check-in would make it into the earlier pull request.

I just wanted to stop this Android Studio preview Exception which was annoying (see screenshot). I don't think the preview will try to animate the image in any way but I could be wrong too.

I'd be happy to change this if you'd like.. or, if there's a better way to submit a pull request so just the specific issue is fixed in 1 commit I can try that too..

thanks,
joe

![image](https://cloud.githubusercontent.com/assets/1895553/22394004/e54bd462-e4d9-11e6-9a2e-8f74fe64b2ff.png)